webapp/sources/rudder/rudder-rest/src/test/resources/api/api_groups.yml

```diff
@@ -74,9 +75,48 @@ response:
       "result":"success",
       "data":{
         "groups":[
-          {
-            "groupId":"0000f5d3-8c61-4d20-88a7-bb947705ba8a",
-            "properties":[
+              {
+                "groupId":"0000f5d3-8c61-4d20-88a7-bb947705ba8a",
+                "properties":[
+              {
+                "name" : "badOverrideType",
+                "value" : {
+                  "now" : "ajson"
+                },
+                "provider" : "overridden",
+                "hierarchy" : [
+                  {
+                    "kind" : "global",
+                    "value" : "astring"
+                  },
+                  {
+                    "kind" : "group",
+                    "name" : "thisgroup",
+                    "id" : "0000f5d3-8c61-4d20-88a7-bb947705ba8a",
+                    "value" : {
+                      "now" : "ajson"
+                    }
+                  }
+                ],
+                "hierarchyStatus" : {
+                  "hasChildTypeConflicts" : true,
```

Comment:
this is the only difference, it could go on the first part and that part be removed